src/test/ui/feature-gates/feature-gate-const_generics.rs

```diff
@@ -1,5 +1,5 @@
-fn foo<const X: ()>() {} //~ ERROR const generics are unstable
```

Comment:
This file can just be entirely deleted now.

Reply:
why?

we still need a feature gate test for full `feature(const_generics)`, don't we?

Reply:
Sorry, misread the file name. Though this was `min_const_generics` 😅